picard/ui/options/scripting.py

```diff
@@ -232,6 +232,14 @@ def __init__(self, parent=None):
         self.last_selected_script_pos = 0
         self.ui.splitter.setStretchFactor(0, 1)
         self.ui.splitter.setStretchFactor(1, 2)
+        self.delete_selected_script = QtWidgets.QShortcut(self.ui.script_list)
+        self.delete_selected_script.setKey(QtGui.QKeySequence.Delete)
+        self.delete_selected_script.activated.connect(self.script_remove_processor)
```

Comment:
@vishichoudhary  I meant the connected function. The shortcut variable itself could be `self.delete_shortcut`.

Reply:
Shouldn't say this but. naming the variables to seem most difficult task for me. :-( 😅